test/test_dist_autograd_fork.py

```diff
@@ -9,9 +9,10 @@
 from dist_autograd_test import DistAutogradTest
 from common_distributed import MultiProcessTestCase
 from common_utils import run_tests
+from process_group_rpc_agent_test_fixture import ProcessGroupRpcAgentTestFixture
 
 
-class DistAutogradTestWithFork(MultiProcessTestCase, DistAutogradTest):
+class DistAutogradTestWithFork(MultiProcessTestCase, ProcessGroupRpcAgentTestFixture, DistAutogradTest):
```

Comment:
Question: if we add this fixture here, how are going to extend this test for other backend?

Reply:
Moving this `RpcAgentTestFixture` to `class DistAutogradTest(RpcAgentTestFixture)`.